print-radius.c

```diff
@@ -671,6 +674,13 @@ print_attr_string(netdissect_options *ndo,
            data++;
            length--;
         break;
+      case EAP_MESSAGE:
+           eap_print(ndo, data);                  
+           ND_TCHECK_2(data + 2);
+           len = EXTRACT_BE_U_2(data + 2);
```

Comment:
Should this length be checked to fit within the current structure before calling `eap_print()`, and should it be passed as an argument to `eap_print()` so it can validate its own structures? (The answers are not necessarily "yes", I did not proof-read every step yet.)